docs-ref-conceptual/authenticate-azure-cli.md

```diff
@@ -34,7 +34,7 @@ After you sign in, CLI commands are run against your default subscription. If yo
 
 ## Refresh tokens
 
-When you sign in with a user account, Azure CLI generates and stores an authentication refresh token. Because access tokens are valid for only a short period of time, a refresh token is issued at the same time the access token is issued. The client application can then exchange this refresh token for a new access token when needed. For more information on token lifetime and expiration, see [Refresh tokens in the Microsoft identity platform](/azure/active-directory/develop/refresh-tokens).
+When you sign in with a user account, Azure CLI generates and stores an authentication refresh token. Because access tokens are valid for only a short period of time, a refresh token is issued at the same time the access token is issued. The client application can then exchange this refresh token for a new access token when needed. Expiration dates are now updated in a format that is supported by [MSAL-based Azure CLI](./msal-based-azure-cli.md). For more information on token lifetime and expiration, see [Refresh tokens in the Microsoft identity platform](/azure/active-directory/develop/refresh-tokens).
```

Comment:
```suggestion
When you sign in with a user account, Azure CLI generates and stores an authentication refresh token. Because access tokens are valid for only a short period of time, a refresh token is issued at the same time the access token is issued. The client application can then exchange this refresh token for a new access token when needed. Use [az account get-access-token](/cli/azure/account#az-account-get-access-token) to retrieve the access token.

Starting from Azure CLI 2.54.0, `az account get-access-token` returns a new `expires_on` property alongside the existing `expiresOn` property for the token expiration time. Find your local datetime in `expiresOn`. The new `expires_on` property represents a Portable Operating System Interface (POSIX) timestamp. This format is supported by [MSAL-based Azure CLI](./msal-based-azure-cli.md). 

We recommend for downstream applications to use the `expires_on` property, because it uses the Universal Time Code (UTC). However, it should be noted that `expiresOn` cannot express "fold" when Daylight Saving Time ends. This can cause problems in countries or regions where Daylight Saving Time is adopted. For more information on "fold", see [PEP 495 – Local Time Disambiguation](https://peps.python.org/pep-0495/).

For more information on token lifetime and expiration, see [Refresh tokens in the Microsoft identity platform](/azure/active-directory/develop/refresh-tokens).
```